packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx

```diff
@@ -30,6 +29,7 @@ export const PastElectionsListRow = ({ election }: PastElectionsListRowProps) =>
         <></>
       )}
       <TokenValue value={election.totalStake} />
```

Comment:
To clarify the code please rename this field:
```suggestion
      <TokenValue value={election.totalCandidatesStake} />
```

Here and in `packages/ui/src/council/types/PastElection.ts` too.